templates/base.html.twig

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
We are using legacy Symfony dir structure, templates should be in `src/Resources/views`.

This one was probably injected by Flex, remove it.